### ISSUE TITLE
list of problems (with meta-data) in readme?

### ISSUE THREAD

**Issue by amueller** [iaroslav-ai/scikit-optimize-benchmarks#4]
It would be great if you could share the list of problems / datasets that you're using for the benchmark in the readme.
Even better would be a table that lists input dimensions and continuous vs discrete parameters and presence of conditional parameters ;)

**Comment by iaroslav-ai** [comment]
Yes, that is something that is missing right now. Currently it is more of a riddle to figure out where things come from :)
I might be slow with issues in this repo right now, but will try to get to this on upcoming holidays latest.
